README.md

```diff
@@ -1,8 +1,34 @@
 # Community Air Quality Dashboard
+This project is a collaboration between the Harvey Mudd College Clinic Program and the South Gate Community Environmental Health Action Team (CEHAT). This repository has all the code for the front and back end of a community air quality monitoring dashboard, accessible at https://sg-cehat-air-quality.web.app. All sensor data is gathered using affordable [PurpleAir](https://www2.purpleair.com) sensors owned by the CEHAT. This website is tailored for the South Gate community, but can easily be used as a template for other communities with the appropriate modifications. One important feature of the website is its availability in both English and Spanish, which is relevant to the community of South Gate, but the langauges can easily be changed.
+
+## Disclaimer
+This repository was worked on by students at Harvey Mudd College from August 2020 until May 2021 and will not be maintained afterwards. It is not likely that this project will be expanded, though it is possible if the CEHAT pursues extensions of this work in the future. Since there are no maintaners of the repository, do not attempt to make any changes, as they will never be reviewed. If you clone this repository for your own work, note that any major changes to the frameworks used in this project will not be reflected in the code.
```

Comment:
```suggestion
## Disclaimer

This repository was worked on by students at Harvey Mudd College from August 2020 until May 2021 and will not be maintained afterwards. It is not likely that this project will be expanded, though it is possible if the CEHAT pursues extensions of this work in the future. Since there are no maintainers of the repository, do not attempt to make any changes or submit any pull requests, as they will never be reviewed. If you clone this repository for your own work, note that any major changes to the frameworks used in this project will not be reflected in the code.
```